release/pkg/bundles/nutanix.go

```diff
@@ -0,0 +1,105 @@
+package bundles
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	anywherev1alpha1 "github.com/aws/eks-anywhere/release/api/v1alpha1"
+	"github.com/aws/eks-anywhere/release/pkg/constants"
+	releasetypes "github.com/aws/eks-anywhere/release/pkg/types"
+	bundleutils "github.com/aws/eks-anywhere/release/pkg/util/bundles"
+	"github.com/aws/eks-anywhere/release/pkg/version"
+)
+
+// GetNutanixBundle returns the bundle for Nutanix
+func GetNutanixBundle(r *releasetypes.ReleaseConfig, eksDReleaseChannel string, imageDigests map[string]string) (anywherev1alpha1.NutanixBundle, error) {
```

Comment:
we don't need to pass eksDReleaseChannel here as there's nothing EKS-D specific here.